js/site.js

```diff
@@ -1,5 +1,15 @@
 // fetch weather data
 
+  let lat;
```

Comment:
I think one issue you're gonna have it `lat` and `long` are not set. In the demo I did with the teens we used the browser tool to get your current position. To start though you can just hard code lat and long. I've done this by going on google maps and searching a location. When you do so the URL should update to something like:

```
https://www.google.com/maps/place/Artists+For+Humanity/@42.3429718,-71.0557969,17z/data=!3m2!4b1!5s0x89e37a7cc30fd4d5:0xa53c3af1827a4aa9!4m5!3m4!1s0x89e37a7cc503b901:0xa35db6dcf24f827e!8m2!3d42.3429718!4d-71.0536082
```

The lat and long from that url is `42.3429718` and `-71.0557969`. Please check the weather api documentation for the format they want the lat and long in.

Reply:
Their docs are [here](https://www.weatherapi.com/docs/).

Reply:
I am now getting an object displayed in the console. I assume I should interpret the data from that object and adjust the circles based on that data?